cmd/soroban-rpc/internal/daemon/daemon.go

```diff
@@ -196,7 +202,20 @@ func MustNew(cfg *config.Config) *Daemon {
 	// NOTE: We could optimize this to avoid unnecessary ingestion calls
 	//       (the range of txmetads can be larger than the store retention windows)
 	//       but it's probably not worth the pain.
+	var initialSeq uint32
+	var currentSeq uint32
 	err = db.NewLedgerReader(dbConn).StreamAllLedgers(readTxMetaCtx, func(txmeta xdr.LedgerCloseMeta) error {
+		currentSeq = txmeta.LedgerSequence()
+		if initialSeq == 0 {
+			initialSeq = currentSeq
+			logger.WithFields(supportlog.F{
+				"seq": currentSeq,
+			}).Info("initializing in-memory store")
+		} else if (currentSeq-initialSeq)%inMemoryInitializationLedgerLogPeriod == 0 {
```

Comment:
this is great to print ongoing status, maybe the period of 1M ledgers does not get reached, is it worthwhile to use a counter that the function closure increments and uses as trigger instead?

Reply:
There are around 40M ledgers in pubnet. I don’t want to flood the logs